src/js/polyfills/focus.js

```diff
@@ -0,0 +1,90 @@
+/* eslint getter-return: 0, no-empty: 0 */
```

Comment:
Did this code come from https://github.com/calvellido/focus-options-polyfill/blob/master/index.js
?

Open-source added to the player needs to be attributed with accordance to it's license. (https://github.com/calvellido/focus-options-polyfill/blob/master/LICENSE)

You'll need to make the appropriate changes in:
- [ ] https://github.com/jwplayer/jwplayer/blob/master/jwplayer.license.notice.js
- [ ] https://github.com/jwplayer/jwplayer-commercial/blob/master/jwplayer.license.notice.js
to include this project in the notice.txt that is output with each build.

Reply:
Thanks!

Reply:
If this did not require any modification then can we bring it via yarn/npm?